### PR TITLE
feat(layout-modules): add `ICAModule`, fix `TitleBarModule` spacing

### PR DESCRIPTION
**Pull request by SimonFinney**
## Proposed changes

- Add `ICAModule` component and stories
- Fix `TitleBarModule` spacing as per design review
- Update ordering of exports to be alphabetical

### CODE REVIEW

File: src/components/UNSTABLE__LayoutModules/ICAModule/index.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+@include security--component($name: #{$layout-modules__name}--ica) {
+  /// Spacing.
+  /// @type Length
+  $ica-module__spacing: $carbon--layout-01;
+
+  padding-top: $ica-module__spacing;
+  padding-bottom: $ica-module__spacing;
+
+  &--hover {
+    padding-top: $ica-module__spacing;
```
Comment (Deriohig) [inline src/components/UNSTABLE__LayoutModules/ICAModule/index.scss:24]: this is probably intentional, but just to be sure the heights of your ICAModule will be different depending if it's interactive or not with these.

**Review by Deriohig — APPROVED** [review]
looks good, one small comment thats probably nothing

File: src/components/UNSTABLE__LayoutModules/ICAModule/index.scss (hunk 1 above)
Comment (SimonFinney) [reply src/components/UNSTABLE__LayoutModules/ICAModule/index.scss:24]: That's exactly right 🎉

**Pull request merged**